role-merge-prs.sh

```diff
@@ -39,33 +39,63 @@ list_prs() {
     gh pr list -R "$origin_org/$repo" "${searchargs[@]}" "$@"
 }
 
-get_check_summary() {
-    local pr total success failure pending cancelled error conclusion state name
+get_checks() {
+    local pr total success failure pending cancelled error conclusion state name status display_name
     pr="$1"
     gh pr view "$pr" -R "$origin_org/$repo" --json statusCheckRollup \
-      --template '{{range .statusCheckRollup}}{{print (or .conclusion "<nil>")}} {{print (or .state "<nil>")}} {{println .name}}{{end}}' | {
-    while read -r conclusion state name; do
+      --template '{{range .statusCheckRollup}}{{print (or .conclusion "<nil>")}}#{{print (or .state "<nil>")}}#{{print .name}}#{{println .context}}{{end}}' | \
+    while IFS="#" read -r conclusion state name context; do
         if [ "$conclusion" = '<nil>' ] && [ "$state" = '<nil>' ]; then
-            pending=$(("${pending:-0}" + 1))
+            status=PENDING
         elif [ "$conclusion" = '<nil>' ] && [ "$state" = PENDING ]; then
-            pending=$(("${pending:-0}" + 1))
+            status=PENDING
         elif [ "$conclusion" = FAILURE ] || [ "$state" = FAILURE ]; then
-            failure=$(("${failure:-0}" + 1))
+            status=FAILURE
         elif [ "$conclusion" = SUCCESS ] || [ "$state" = SUCCESS ]; then
-            success=$(("${success:-0}" + 1))
+            status=SUCCESS
         elif [ "$conclusion" = CANCELLED ] || [ "$state" = CANCELLED ]; then
-            cancelled=$(("${cancelled:-0}" + 1))
+            status=CANCELLED
         elif [ "$conclusion" = ERROR ] || [ "$state" = ERROR ]; then
-            error=$(("${error:-0}" + 1))
+            status=ERROR
         elif [ "$conclusion" = NEUTRAL ]; then  # usually a python check if python code did not change
-            success=$(("${success:-0}" + 1))
+            status=SUCCESS
         else
             echo ERROR: check "$name" has unknown conclusion "$conclusion" state "$state" 1>&2
             exit 1
         fi
+        if [ "$name" != '<nil>' ]; then
+            display_name="$name"
+        elif [ "$context" != '<nil>' ]; then
+            display_name="$context"
+        fi
+        echo "${display_name}#${status}"
+    done
+}
+
+get_check_detail() {
+    local pr name status
+    pr="$1"; shift
+    get_checks "$pr" | while IFS=# read -r name status; do
```

Comment:
```suggestion
    get_checks "$pr" | while IFS="#" read -r name status; do
```
You do this above, with quotes it's better